mailpoet/lib/Mailer/MailerLog.php

```diff
@@ -4,25 +4,53 @@
 
 use MailPoet\Settings\SettingsController;
 
+/**
+ * @phpstan-type MailerLogError array{
+ *    "error_code"?: non-empty-string,
+ *    "error_message": string,
+ *    "operation": string
+ *   }
+ * @phpstan-type MailerLogData array{
+ *   "sent": array<string,int>,
+ *   "started": int,
+ *   "status": ?string,
+ *   "retry_attempt": ?int,
+ *   "retry_at": ?int,
+ *   "error": ?MailerLogError
+ * }
+ */
+
 class MailerLog {
   const SETTING_NAME = 'mta_log';
   const STATUS_PAUSED = 'paused';
   const RETRY_ATTEMPTS_LIMIT = 3;
   const RETRY_INTERVAL = 120; // seconds
 
-  public static function getMailerLog($mailerLog = false) {
+  /**
+   * @param MailerLogData|null $mailerLog
+   * @return MailerLogData
+   */
+  public static function getMailerLog(array $mailerLog = null): array {
     if ($mailerLog) return $mailerLog;
     $settings = SettingsController::getInstance();
     $mailerLog = $settings->get(self::SETTING_NAME);
     if (!$mailerLog) {
       $mailerLog = self::createMailerLog();
     }
+    /**
+     * The old "sent" entry was just the number of emails.
+     * We need to update this entry to the new data structure.
+     */
+    $mailerLog['sent'] = is_numeric($mailerLog['sent']) ? [self::sentEntriesDate(time() - 1) => $mailerLog['sent']] : (array)$mailerLog['sent'];
```

Comment:
Hello @rodrigoprimo,
thanks! I forgot to check that and it slipped through. I address it in 98f50e6b7

Reply:
Thanks, @websupporter. I can confirm that the page is now working. I wonder if there are other places that rely on the format of `$mailLog['sent']` that could be broken as well? I don't recall anything off the top of my head, but maybe worth checking in case you haven't already?